server/document.js

```diff
@@ -104,12 +104,14 @@ export class Head extends Component {
     const { page, pathname, buildId } = __NEXT_DATA__
     const pagePathname = getPagePathname(pathname)
 
+    let children = this.props.children
     // show a warning if Head contains <title> (only in development)
     if (process.env.NODE_ENV === 'development') {
-      React.Children.forEach(this.props.children, (child) => {
+      children = React.Children.map(children, (child) => {
```

Comment:
Hey @lucleray 
Thanks for your nice job! Just one question:
What was the reason to move from `React.Children.forEach` to `React.Children.map`?

Reply:
Because of this comment on the other PR : https://github.com/zeit/next.js/pull/4631#discussion_r217664081